app/src/main/res/drawable/baseline_circle_24.xml

```diff
@@ -0,0 +1,5 @@
+<vector android:height="36dp" android:tint="#000000"
```

Comment:
If our marker is a simple circle, and we're already drawing other circles directly onto the `Canvas` of our markers, do we really need a separate vector graphic of a circle?

Reply:
Sure! Done!